.env.example

```diff
@@ -0,0 +1,15 @@
+# SP1_PROVER={network|local|mock}
```

Comment:
[no change needed] should we create a follow-up to de-duplicate this with the .env file that is created at `./solidity-ibc-eureka/.env` via the `setup-env.go`